src/main/build.gradle

```diff
@@ -27,4 +27,8 @@ dependencies {
     implementation 'org.json:json:20201115'
 }
 
+jar {
+    archiveBaseName = 'Amplitude-Java'
+    archiveVersion =  '1.0.0'
+}
 jar.baseName = 'Amplitude-Java'
```

Comment:
Is jar.baseName duplicate with jar {archiveBaseName XXX} ? 
I tested the jar.baseName =  'Amplitude-Java'. It will make the jar name to `Amplitude-Java`.

Reply:
Good catch. jar.baseName is deprecated. This does the same function. I just found documentation [here](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html).